src/Data/List/Relation/Unary/All/Properties.agda

```diff
@@ -385,6 +388,11 @@ concat⁻ : ∀ {xss} → All P (concat xss) → All (All P) xss
 concat⁻ {xss = []}       []  = []
 concat⁻ {xss = xs ∷ xss} pxs = ++⁻ˡ xs pxs ∷ concat⁻ (++⁻ʳ xs pxs)
 
+all⊆concat : (xss : List (List A)) → All (Sublist._⊆ (concat xss)) xss
```

Comment:
Very minor but we don't need the brackets around `concat xss` right?

Reply:
Done.